Makefile

```diff
@@ -178,7 +178,7 @@ SELPROJ = $(LOCALBIN)/selproj-$(SELPROJ_VERSION)
 
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v4.5.7
-CONTROLLER_TOOLS_VERSION ?= v0.10.0
+CONTROLLER_TOOLS_VERSION ?= v0.15.0
```

Comment:
Updating controller-gen is a rather large change, why not do this in a separate PR?